src/plugins/console/tsconfig.json

```diff
@@ -2,11 +2,7 @@
   "extends": "../../../tsconfig.base.json",
   "compilerOptions": {
     "outDir": "target/types",
-    // there is still a decent amount of JS in this plugin and we are taking
-    // advantage of the fact that TS doesn't know the types of that code and
-    // gives us `any`. Once that code is converted to .ts we can remove this
-    // and allow TS to infer types from any JS file imported.
-    "allowJs": false
+    "allowJs": true,
```

Comment:
I think we can simply the  `allowJS` property as it's set to `true` in the file `tsconfig.base.json`.
```suggestion
```